config/cly/cly_config.ini

```diff
@@ -7,139 +7,105 @@
     "n200s" : [
         {
             "addr" : "192.168.10.100",
-            "rx" : true,
-            "tx" : true,
-            "rx_int" : false,
-            "main_antenna" : "0",
-            "intf_antenna" : ""
+            "rx_channel_0" : "m0",
+            "rx_channel_1" : "",
+            "tx_channel_0" : "m0"
```

Comment:
This is a much cleaner and clearer way of configuring the N200s, I'm glad we came to this decision

Reply:
As am I. It was much easier to modify as well, no mental gymnastics required at all.